ResearchStack2/ResearchStack2Tests/Codable Tests/CodableDistanceRecorderTests.swift

```diff
@@ -38,7 +38,10 @@ class CodableDistanceRecorderTests: XCTestCase {
     
     override func setUp() {
         super.setUp()
-        // Put setup code here. This method is called before the invocation of each test method in the class.
+        
+        // Use a statically defined timezone that is **not** a real timezone to check that
```

Comment:
The people of Newfoundland and southeastern Labrador might take issue with characterizing their time zone as "not real." ;)

FWIW in the BridgeSDK integration tests I do timezone-dependent tests three times, once in GMT-something (Pacific time I think), once in GMT, and once in GMT+something (Hong Kong iirc). It might be overkill but I've actually caught a couple of server-side time zone bugs this way when certain of the tests would fail at certain times of day and work at other times.

Reply:
seriously? oops. 

This is just testing the Codable protocol and **not** checking against a service, so I don't think multiple timezones is necessary for any of these tests.